### PR TITLE
Turn Indicator: Using GameBoard state to display which players' turn it is

### PR DESCRIPTION
**Pull request by debughug**
**Screenshot of Change**
image

### CODE REVIEW

**Comment by debughug** [comment]
Noticed you're using 4 space indents, would you like me to update to your rules or follow a 2 space indentation?

File: src/Components/GameBoard.js [hunk 1]
```diff
@@ -1,74 +1,63 @@
...
+      return;
     }
+  }
+
+  render() {
+    return (
+      <div className="board-wrapper">
+        <div className="hud">
+          <h3 className="turn-indicator">Current Player: {this.state.xIsNext ? 'X' : 'O'}</h3>
+        </div>
```
Comment (debughug) [inline src/Components/GameBoard.js:42]: Placed HUD above the board. Currently uses the state's xIsNext value

File: src/Styles/GameBoard.css [hunk 2]
```diff
@@ -1,13 +1,18 @@
-
 .board-wrapper {
-    display: flex;
-    justify-content: center;
+  align-items: center;
+  display: flex;
+  flex-direction: column;
+  justify-content: center;
```
Comment (debughug) [inline src/Styles/GameBoard.css:5]: Updated board's flex rules so it would stack HUD above the board

File: src/Styles/GameBoard.css [hunk 3]
```diff
@@ -1,13 +1,18 @@
...
 }
 
 button {
-    font-size: 100px;
+  font-size: 100px;
+}
+
+.hud {
+  margin-bottom: 25px;
 }
```
Comment (debughug) [inline src/Styles/GameBoard.css:18]: HUD and board spacing by 25px - Procore's magic spacing base value - I agree with the amount so just copying them.

File: src/index.css [hunk 4]
```diff
@@ -1,13 +1,35 @@
...
+  font-size: 30px;
+}
+
+h4 {
+  font-size: 24px;
+}
+
+h5 {
+  font-size: 20px;
 }
```
Comment (debughug) [inline src/index.css:35]: Place font sizes for heading tags and global

**Comment by christulin** [comment]
> Noticed you're using 4 space indents, would you like me to update to your rules or follow a 2 space indentation? You'll see some changes are just indentation updates and not actual code.



> Noticed you're using 4 space indents, would you like me to update to your rules or follow a 2 space indentation? You'll see some changes are just indentation updates and not actual code.

I'm cool to use 2 space indentation as standard

**Pull request merged**